src/Support/RateLimit.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Spatie\Ray\Support;
+
+class RateLimit
+{
+    /** @var int|null */
+    protected static $maxCalls;
+
+    /** @var int|null */
+    protected static $maxPerSecond;
+
+    /** @var CacheStore */
+    protected static $cache;
+
+    private function __construct(?int $maxCalls, ?int $maxPerSecond)
+    {
+        self::$maxCalls = $maxCalls;
```

Comment:
Do these things need to be static? Don't we already keep the entire instance statically in Ray?